model/SyncLog/Storage/RdsSyncLogStorage.php

```diff
@@ -0,0 +1,237 @@
+<?php
+ /**
+  * This program is free software; you can redistribute it and/or
+  * modify it under the terms of the GNU General Public License
+  * as published by the Free Software Foundation; under version 2
+  * of the License (non-upgradable).
+  *
+  * This program is distributed in the hope that it will be useful,
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  * GNU General Public License for more details.
+  *
+  * You should have received a copy of the GNU General Public License
+  * along with this program; if not, write to the Free Software
+  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+  *
+  * Copyright (c) 2018 (original work) Open Assessment Technologies SA ;
+  */
+
+ namespace oat\taoSync\model\SyncLog\Storage;
+
+ use Doctrine\DBAL\Query\QueryBuilder;
+ use oat\oatbox\service\exception\InvalidServiceManagerException;
+ use oat\oatbox\extension\script\MissingOptionException;
+ use oat\oatbox\service\ConfigurableService;
+ use oat\taoSync\model\SyncLog\SyncLogEntity;
+ use oat\taoSync\model\SyncLog\SyncLogFilter;
+ use common_exception_NotFound;
+ use common_persistence_SqlPersistence;
+ use Doctrine\DBAL\Connection;
+
+ class RdsSyncLogStorage extends ConfigurableService implements SyncLogStorageInterface
+ {
+     const SERVICE_ID = 'taoSync/RdsSyncLogStorage';
+
+     const OPTION_PERSISTENCE_ID = 'persistenceId';
+
+     const TABLE_NAME = 'synchronisation_log';
+
+     /**
+      * @var string
+      */
+     private $persistenceId = null;
+
+     /**
+      * @var common_persistence_SqlPersistence
+      */
+     private $persistence = null;
+
+     /**
+      * SyncLogService constructor.
+      * @param array $options
+      * @throws MissingOptionException
+      */
+     public function __construct(array $options)
+     {
+         parent::__construct($options);
+
+         if (!$this->hasOption(self::OPTION_PERSISTENCE_ID)) {
+             throw new MissingOptionException('Persistence ID is required option.', self::OPTION_PERSISTENCE_ID);
+         }
+         $this->persistenceId = $this->getOption(self::OPTION_PERSISTENCE_ID);
+     }
+
+     /**
+      * @return common_persistence_SqlPersistence
+      */
+     public function getPersistence()
+     {
+         if ($this->persistence === null) {
+             $this->persistence = $this->getServiceLocator()
+                 ->get(\common_persistence_Manager::SERVICE_ID)
+                 ->getPersistenceById($this->persistenceId);
+         }
+
+         return $this->persistence;
+     }
+
+     /**
+      * @return QueryBuilder
+      */
+     private function getQueryBuilder()
+     {
+         return $this->getPersistence()->getPlatForm()->getQueryBuilder()->from(self::TABLE_NAME);
+     }
+
+     /**
+      * Store SyncLogEntity in rds storage.
+      *
+      * @param SyncLogEntity $entity
+      * @return integer
```

Comment:
it would be better to return boolean value

Reply:
Fixed annotations to return integer (id of created record)